site/content/en/docs/manual/advanced/annotation-with-skeletons/_index.md

```diff
@@ -2,15 +2,23 @@
 title: 'Annotation with skeletons'
 linkTitle: 'Annotation with skeletons'
 weight: 12
-description: 'Guide to creating and editing skeletons.'
+description: 'Guide to creating and editing skeletons'
 ---
 
 
-Skeletons should be used as annotations `templates` when you need to annotate complex objects sharing the same structure
-(e.g. human pose estimation, facial landmarks, etc.).
-A skeleton consist of any number of points (also called as elements), joined or not joined by edges.
-Any point itself is considered like an individual object with its own attributes and properties
-(like color, occluded, outside, etc). At the same time a skeleton point can exist only within the parent skeleton.
+**Skeletons** serve as annotation templates
+for annotating complex objects with a consistent structure,
+such as human pose estimation or facial landmarks.
 
-Any skeleton elements can be hidden (by marking them `outside`) if necessary (for example if a part is out of a frame).
-Currently there are two formats which support exporting skeletons: CVAT & COCO.
+A **Skeleton** is composed of numerous points (also referred to as elements),
+which may be connected by edges. Each point functions as an individual object,
+possessing unique attributes and properties like color, occlusion, and visibility.
+
+However, a **Skeleton** point can only exist within its parent **Skeleton**.
+
+**Skeleton** elements can be marked as `outside` and hidden if needed,
+such as when a part is outside the frame.
```

Comment:
`outside` is also skeleton property like occlusion or color. It is not totally clear why it mentioned after one paragraph. 

Also there is some mess between visibility, occluded and outside. Maybe to clarify that in details. 
Both outside and visibility may do a skeleton point invisible. But outside property is part of annotations, when visibility makes a point hidden only for annotator's convenience and not saved somehow between different sessions.

Occluded and outside are similar in their meaning, but `occluded` keeps point visible on the frame and usually means that point still on a frame, just hidden by some upper object. Outside is recommended to be used when part of object is out of frame borders.